EIPS/eip-9999.md

```diff
@@ -0,0 +1,191 @@
+---
+eip: 9999
+title: Static relative jumps and calls for the EVM
+description: Relative jump and call instructions with a signed immediate encoding the jump destination
+author: Greg Colvin (@gcolvin), Alex Beregszaszi (@axic), Andrei Maiboroda (@gumb0), Paweł Bylica (@chfast)
+discussions-to: https://ethereum-magicians.org/t/eip-0000-static-relative-jumps-and-calls/0000
```

Comment:
Please create a discussions topic on Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5